### PR TITLE
Missing API docs part 1 - SA

### PR DESCRIPTION
**Pull request by steveada**
Totals 51 rows over 12 files

### CODE REVIEW

**Comment by steveada** [comment]
@mairaw Including you on review

**Comment by mairaw** [comment]
Build failed with a weird error message. Trying to build again.

**Comment by steveada** [comment]
@mairaw Yes, I have some unescaped characters in one of the files; fixing them now

File: xml/Microsoft.VisualBasic.Vsa/VsaGlobalItem.xml [hunk 1]
```diff
@@ -76,8 +76,8 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets or sets the type of the global item.</summary>
```
Comment (mairaw) [inline xml/Microsoft.VisualBasic.Vsa/VsaGlobalItem.xml]:
```suggestion
        <summary>Sets the type of the global item.</summary>
```

File: xml/Microsoft.VisualBasic.Vsa/VsaGlobalItem.xml [hunk 2]
```diff
@@ -76,8 +76,8 @@
...
+        <value>The item type of the global item.</value>
```
Comment (mairaw) [inline xml/Microsoft.VisualBasic.Vsa/VsaGlobalItem.xml]:
```suggestion
        <value>The type of the global item.</value>
```

File: xml/Microsoft.Build.Tasks/GenerateManifestBase.xml [hunk 3]
```diff
@@ -289,11 +289,12 @@
         <Parameter Name="versionB" Type="System.String" Index="1" FrameworkAlternate="netframework-3.5" />
       </Parameters>
       <Docs>
-        <param name="versionA">To be added.</param>
-        <param name="versionB">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="versionA">A string containing the major and minor versions of the first framework to compare, and also optionally build and revision numbers, where each number is delimited with a period character ('.').</param>
```
[the remark this answers is not in the document]
Comment (steveada) [reply xml/Microsoft.Build.Tasks/GenerateManifestBase.xml]: Will do. Still, my guesses weren't too bad :)

**Pull request merged**